clients/web/src/views/body/PluginsView.js

```diff
@@ -127,7 +128,22 @@ var PluginsView = View.extend({
                   }
               }
               $('button.g-rebuild-and-restart').addClass('btn-danger');
-              $('.g-plugin-rebuild-restart-text').addClass('show');
+
+              if (view.cherrypyServer) {
+                  $('.g-plugin-rebuild-restart-text').addClass('show');
+              }
+
+              if (!view.cherrypyServer && !_.has(view, 'displayedCherrypyNotification')) {
```

Comment:
Probably worthwhile to convert this to an arrow function and ditch the `view` variable.